tag_dedupe/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM gcr.io/cloud-builders/gcloud
+
+COPY dedupe.py /
+
+# This is the file containing the auth information for your service account. 
+# Necessary inside the container until gcloud supports cross container authentication when using cloudbuild.
+# Once this is addressed, remove this and update dedupe.py accordingly.
+COPY auth.json /
```

Comment:
Actually, you could probably run:

`RUN /builder/google-cloud-sdk/bin/gcloud auth activate-service-account /auth.json` here, and then take it out of your python code.

Then once the feature starts working you can just delete these two lines.